### PR TITLE
Provide API key to crawler

### PR DESCRIPTION
**Pull request by DerekRoberts**
---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://public-code-101-frontend.apps.silver.devops.gov.bc.ca/) available
[Backend](https://public-code-101-backend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/public-code/actions/workflows/merge-main.yml)

### CODE REVIEW

File: .github/workflows/pubcode-crawler-on-scheduler.yml [hunk 1]
```diff
@@ -40,6 +40,10 @@ jobs:
...
         env:
           GIT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          API_KEY: ${{ secrets.OC_PROJECT}}
         run: |
-          node src/main.js
+          oc login --token=${{ secrets.OC_TOKEN }} --server=${{ secrets.OC_SERVER }}
+          oc project ${{ secrets.OC_NAMESPACE }}
+
+          # Get API key
+          API_KEY=$(oc get secrets/public-code-prod-backend --template={{.data.api_key}} | base64 -d)
```
Comment (mishraomp) [inline .github/workflows/pubcode-crawler-on-scheduler.yml:48]: I believe it needs to be exported to PATH to be injected as env var

Comment (DerekRoberts) [reply .github/workflows/pubcode-crawler-on-scheduler.yml:48]: That's what the following line does.  It's provided while running node.  We can change if you prefer a different format.

**Review by mishraomp — APPROVED** [review]
LGTM

File: .github/workflows/pubcode-crawler-on-scheduler.yml (hunk 1 above)
Comment (DerekRoberts) [reply .github/workflows/pubcode-crawler-on-scheduler.yml:48]: E.g. `THING=thing node -e "console.log(process.env.THING)"`

**Pull request merged**